keylime-push-model-agent/src/struct_filler.rs

```diff
@@ -48,6 +48,17 @@ impl StructureFiller for FillerFromHardware<'_> {
     }
 }
 
+fn boot_time() -> chrono::DateTime<chrono::Utc> {
+    match keylime::boot_time::get_boot_time() {
+        Ok(time) => time,
+        Err(e) => {
+            error!("Failed to get boot time: {e}");
```

Comment:
Could we make this message a warning? And also include the information that we are falling back to EPOCH?